src/server/transforms.js

```diff
@@ -13,7 +13,8 @@ function baseData(item, options, callback) {
     language: 'javascript',
     framework: item.framework || options.framework,
     uuid: item.uuid,
-    notifier: JSON.parse(JSON.stringify(options.notifier))
+    notifier: JSON.parse(JSON.stringify(options.notifier)),
+    custom: item.custom
```

Comment:
This results in duplicated properties in Occurrences view... Could we revert this part?

Reply:
I think the intent is to support the `custom` key as defined by the API. https://explorer.docs.rollbar.com/#operation/create-item

Maybe a config option for this is the best way to meet both needs.